src/callbacks.c

```diff
@@ -1279,10 +1271,10 @@ G_MODULE_EXPORT void on_print1_activate(GtkMenuItem *menuitem, gpointer user_dat
 
 G_MODULE_EXPORT void on_menu_select_all1_activate(GtkMenuItem *menuitem, gpointer user_data)
 {
-	GeanyDocument *doc = document_get_current();
-	g_return_if_fail(doc != NULL);
+	GtkWidget *focusw = gtk_window_get_focus(GTK_WINDOW(main_widgets.window));
+	g_return_if_fail(IS_SCINTILLA(focusw));
 
-	sci_select_all(doc->editor->sci);
+	sci_select_all(SCINTILLA(focusw));
```

Comment:
Hum, this callback is odd, and probably should rather call `GEANY_KEYS_SELECT_ALL` to support everything… or maybe the other way around, for this to support everything and be called by `cb_func_select_action()`.

Though, this is a mostly separate issue.